rules/cloud/github/github_excessive_repository_clones.yml

```diff
@@ -0,0 +1,28 @@
+title: Excessive GitHub Repository Clones by a Single User
+id: 9418984c-2e0e-4312-b1a6-7847a85e58e0
+status: test
+description: Detects when a user clones more than 10 different repositories within an hour, which may indicate potential data exfiltration or unauthorized activities.
+author: Ivan Saakov
+date: 2024-09-20
+references:
+  - https://docs.github.com/en/organizations/keeping-your-organization-secure/managing-security-settings-for-your-organization/audit-log-events-for-your-organization
+tags:
+  - attack.exfiltration
+  - attack.t1020
+logsource:
+  product: github
+  service: audit
+detection:
+  selection:
+    action: 'git.clone'
+  timeframe: 1h
+  condition: selection
+  aggregation:
+    group-by: user
+    cardinality:
+      field: repo
+    op: '>='
+    value: 10
```

Comment:
Correlation rules aren't yet supported in the SigmaHQ repo. So please refrain from contributing them for now.

Reply:
> Correlation rules aren't yet supported in the SigmaHQ repo. So please refrain from contributing them for now.

thanks for the information, I will keep this in mind in the future